internal/clients/http/sources/sources_client_test.go

```diff
@@ -74,3 +74,71 @@ func TestSourcesClient_GetAuthentication(t *testing.T) {
+			require.NoError(t, err, "failed to write http body for stubbed server")
+		})
```

Comment:
The response is huge, should I extract the responses to a different file?
 maybe create a fixtures folder for those responses.

Reply:
Good point, I think this passed the readability threshold for me and I would appreciate separate files for all cases, ideally with JSON-formatted content so we can easily track diffs in git when Sources update their API.